python/cxx/group.cpp

```diff
@@ -8,18 +8,13 @@ namespace {
   const std::vector<std::string> wellnames( const Group& g ) {
         return g.wells( );
     }
-
-     int get_vfp_table_nr( const Group& g ) {
-        return g.getGroupNetVFPTable();
-    }
 }
 
 void python::common::export_Group(py::module& module) {
 
   py::class_< Group >( module, "Group")
     .def_property_readonly( "name", &Group::name)
-    .def_property_readonly( "num_wells", &Group::numWells)     
-    .def( "_vfp_table_nr", &get_vfp_table_nr )
+    .def_property_readonly( "num_wells", &Group::numWells)
```

Comment:
I could not see what is changed in this line.

Reply:
That is also what I thought. But I thought there might be a light color to show the difference.

Reply:
there is not, but you can select with the mouse to see it.